client/src/components/settings/Certification.js

```diff
@@ -482,8 +482,8 @@ export class CertificationSettings extends Component {
       is2018DataVisCert &&
       isApisMicroservicesCert &&
       isFrontEndLibsCert &&
-      isQaCert &&
-      isInfosecCert &&
+      is2020QaCert &&
+      is2020InfosecCert &&
       isJsAlgoDataStructCert &&
       isRespWebDesignCert;
```

Comment:
This block does not have the new python certs

Reply:
Is the plan to make Full Stack include the three Python certs or are they going to be separate?

Reply:
IDK yet. I left an open issue to discuss with the team. 

Other than that this is a straight up /s/isCert/is2020Cert/g

Reply:
So, the current full stack will move into a legacy section.